src/System.IdentityModel.Tokens.Jwt/JwtHeader.cs

```diff
@@ -111,6 +127,20 @@ public JwtHeader(SigningCredentials signingCredentials, IDictionary<string,strin
         /// <param name="outboundAlgorithmMap">provides a mapping for the 'alg' value so that values are within the JWT namespace.</param>
         /// <exception cref="ArgumentNullException">If 'signingCredentials' is null.</exception>
         public JwtHeader(EncryptingCredentials encryptingCredentials, IDictionary<string, string> outboundAlgorithmMap)
+            : this(encryptingCredentials, outboundAlgorithmMap, null)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="JwtHeader"/>.
+        /// With the Header Parameters:
+        /// <para>{ { typ, JWT }, { alg, SigningCredentials.Algorithm } }</para>
+        /// </summary>
+        /// <param name="encryptingCredentials"><see cref="EncryptingCredentials"/> used when creating a JWS Compact JSON.</param>
+        /// <param name="outboundAlgorithmMap">provides a mapping for the 'alg' value so that values are within the JWT namespace.</param>
+        /// <param name="tokenType"> provides the token type</param>
+        /// <exception cref="ArgumentNullException">If 'signingCredentials' is null.</exception>
```

Comment:
should be if 'encryptingCredentials' is null

Reply:
Good catch, went unnoticed in previous releases 😆